client/scripts/build-modules.js

```diff
@@ -0,0 +1,186 @@
+const { build, cliopts, basename, stdoutStyle: styl, fmtDuration } = require("estrella")
```

Comment:
That would be fine, I suppose. It makes sense. However, in the `root` package, I don't want the `package.json` to use `type: "module"` because that sometimes does some _Really Wacky Behavior_ with build tools.

Reply:
True. I take this back, no reason to play with fire unless there's a great need to. I might consider swapping the .cjs for .js at some point, lmao